src/index.html

```diff
@@ -15,18 +16,50 @@
       rel="stylesheet"
       href="style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <!-- #endregion -->
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="form">
+      <div
+        class="form__search"
+        data-qa="big"
+      >
+        <label for="input-large"></label>
+        <img
+          class="form-img form__img--large"
+          src="images/Search.svg"
+          alt="Search"
+        />
+        <input
+          class="form-input form__input--large"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <div
+        class="form__search"
+        data-qa="small"
+      >
+        <label for="input-small"></label>
```

Comment:
Attribute 'for' in the label should match the ID of the input it is related to, which is missing here.